modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,7 +33,50 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
 
 You can specify the version of the proxy you want to use by clicking the *Advanced Options* drop-down menu and selecting the proxy version that best covers your needs.
 
-See xref:proxy-deploy-raml-oas-proxy.adoc[Building a RAML/OAS API Proxy] for more information.
+== API Proxy Validations
+
+With Mule 4, you can perform strict validations directly from API Manager. Strict validations enable the application to compare a request with proxy specifications and allow only those headers and query parameters that are specified in the API definition. 
```

Comment:
This should not be the intro of the whole feature. It only talks about mule 4 strict headers. I would add an intro talking about the feature in general, saying all the runtimes that support, and the types of proxies that are supported (raml/rest and wsdl).
Also, IMHO, I would then include the default values, and finally I would include this whole text, which only talks about the specific scenarios when using mule 4 and strict validations.